config/database.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: off */
 module.exports = ({ env }) => {
   const connection = env('CONNECTION', 'default');
   console.log(`Current Connection: ${connection}`);
```

Comment:
antar du fjerner denne senere?

Reply:
Den er tiltenkt for å bli, siden vi vil vite hvilken `CONNECTION` som brukes.